main.js

```diff
@@ -0,0 +1,90 @@
+
+// Task #1
+
+// document.querySelector('#first-btn').onclick = transformToСentury;
+// const outFirstTask = document.querySelector('.first-task');
+
+// function transformToСentury() {

```

Comment:
It works with strings

<img width="336" alt="image" src="https://user-images.githubusercontent.com/84010501/200805503-cfdfe43f-64b1-4fd1-85ba-2af0af5bf4b6.png">

Century range should start from 1st
<img width="336" alt="image" src="https://user-images.githubusercontent.com/84010501/200806672-60e315a6-2b98-4d8c-a5cf-4172689db38e.png">